docs/antimony.rst

```diff
@@ -1393,4 +1425,4 @@ Further Reading
 ---------------
 
 * Lucian Smith's `example models <http://antimony.sourceforge.net/antimony-examples.html>`_ show how to use the `comp package <http://sbml.org/Documents/Specifications/SBML_Level_3/Packages/comp>`_.
-* `Antimony's manual <http://antimony.sourceforge.net/Tutorial.pdf>`_ in PDF format.
```

Comment:
I don't know why a diff is shown here since I didn't modify anything here. The two lines seem identical.

Reply:
It looks like the 'Antimony's manual' line was removed.  Probably accidentally?

Reply:
the same line is actually added below this line, under this conversation window. The window is a bit unfortunately placed.